### DISCUSSION TITLE
Cybersecurity Superstition

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#17] [Comments Archive]
# Cybersecurity Superstition

A look into some cybersecurity myths and misinformation. Explore the best practices regarding passwords, multifactor authentication, social engineering tactics, antiviruses, and much more.

https://vale.rocks/posts/cybersecurity-superstition

**Comment by DeclanChidlow** [comment]
I tried to write this article in such a way that it wouldn't be a slap in the face to my usual audience while still remaining informative for the less technically inclined. I'm worried I underestimated just how technically inept the population is...

> "Turn off your computer at night. So they can't get into it"...

https://abc7chicago.com/remote-access-trojan-horse-virus-rat-attack-computer-hacking/14479523

**Comment by DeclanChidlow** [comment]
Hacker News served me an excellent article from [The New Oil](https://thenewoil.org) today that more or less perfectly reflects my view on antiviruses.

You can read it here:
https://blog.thenewoil.org/do-you-even-need-antivirus-software-in-2024

**Comment by osckig** [comment]
I would like to get into programming but finding it hard, any tips?

**Reply by DeclanChidlow** [reply]
I think the best way to get into programming is to jump in head first. Just start doing things. Yes, you'll be bodging things, and they'll be terrible, but you'll start learning. Don't be afraid to break stuff. Make backups, and employ some sort of version control like Git, and you'll be good to go.

If you're a student, then check out [GitHub Education](https://education.github.com). There is a lot of very helpful and free stuff there.

**Reply by osckig** [reply]
Ok, thanks. I'm going on Github Education right now. 
Also, any advice on data structures and algorithms?

**Reply by DeclanChidlow** [reply]
Can't say I've got much specific advice to give regarding data structures and algorithms. Depending on where you are in your journey, Harvard's CS50 course could be pretty valuable for understanding some of the more core concepts.

**Reply by osckig** [reply]
Hi there, I've been going over bezier curves and also how I can implement them using a programming language. Very interesting stuff.
